examples/looker_embed_generator/README.md

```diff
@@ -19,13 +19,16 @@ These are command line applications. `LOOKERKEY` must be set for this session.
 Parameters:
 - `env`: the target Looker environment, e.g.: `prod` or `test`
 - `embed_url`: the target look or dashboard, e.g.: `looks/98`
+- `-e`: set the flag for optional embed filter
+- `embed filter jsons string`: a JSON string specifying the filter, e.g.: '{"filterName":"City","matchType":"=","matchValue":"Metropolis"}'
+- `-u`: set the flag for optional user-attribute
 - `attribute`: a User-Attribute to pass, e.g.: `browser`
 - `filter`: a filter on the passed User-Attribute, e.g.: `Chrome`
 
 The Java version accepts all optional attribute and filter parameters as follows:
 
 ```
-java looker_embed_generator <<environment>> <<embed_url>> [<<attribute>> <<filter>>]
+java looker_embed_generator <<environment>> -e  '{"filter-name":"filtername-value","matchtype":"matchtype-value","values":"filter-value"}' -u <<embed_url>> [<<attribute>> <<filter>>]
```

Comment:
Since the gson requirement, this no longer works. Please provide an example where the `-classpath` or `-cp` is set and the gson requirement is passed to it.

Reply:
Updated the example commands.